src/main/kotlin/com/hyperdevs/poeditor/gradle/xml/AndroidXmlWriter.kt

```diff
@@ -61,13 +63,14 @@ class AndroidXmlWriter {
                 baseValuesDir
             }
         }
+        val resourceFileName = resFileName.ifEmpty { "strings" }
```

Comment:
You shouldn't do this here: you have to add a new `convention` in `PoEditorPlugin`'s `mainPoEditorExtension` creation where you can set the default value for the new property. Here are the relevant lines where you should add the new property `convention`:
https://github.com/hyperdevs-team/poeditor-android-gradle-plugin/blob/f40d6b703edb9fcf33aefccafc19d4e55ee3adb2/src/main/kotlin/com/hyperdevs/poeditor/gradle/PoEditorPlugin.kt#L52

Reply:
Ah, completely missed this. Of course, this is much better! Fixed 👍